website-hugo/src/themes/covid19/layouts/shortcodes/uploadform.html

```diff
@@ -1,106 +1,113 @@
-    </div>
 
-    <div class="question">
-        <label for="symptomsDate">{{ i18n "when_first_symptoms" }}</label>
-        <div class="options">
-            <input type="date" id="symptomsDate" name="symptomsDate" value="">
+        <div class="question">
+            <label for="symptomsDate">{{ i18n "when_first_symptoms" }}</label>
+            <div class="options">
+                <input type="date" id="symptomsDate" name="symptomsDate" value="">
+            </div>
         </div>
-    </div>
 
-    <div class="question">
-        <label for="contactoptions">{{ i18n "in_contact_with_positive_covid" }}</label>
-        <div id="contactoptions" class="options">
-            <input type="radio" id="contactyes" name="contact" value="yes">
-            <label for="contactyes">{{ i18n "yes" }}</label>
-            <input type="radio" id="contactno" name="contact" value="no">
-            <label for="contactno">{{ i18n "no" }}</label>
+        <div class="question">
+            <label for="contactoptions">{{ i18n "in_contact_with_positive_covid" }}</label>
+            <div id="contactoptions" class="options">
+                <input type="radio" id="contactyes" name="contact" value="yes">
+                <label for="contactyes">{{ i18n "yes" }}</label>
+                <input type="radio" id="contactno" name="contact" value="no">
+                <label for="contactno">{{ i18n "no" }}</label>
+            </div>
         </div>
     </div>
-</div>
 
-<div class="questionblock">
-    <label>
-        Google Location History <span class="small">(<a href="{{ .Site.LanguagePrefix }}/howto" target="_blank">{{ i18n "see_step_guide" }}</a>)</span>
-    </label>
-    <div class="question">
-        <label for="file">{{ i18n "select_location_file" }} <b>Location History.json</b></label>
-        <div class="options">
-            <input class="form-control" type="file" id="file" name="file">
+    <div class="questionblock">
+        <h2>Location History</h2>
+        <label>
+            Use your <a href="{{ .Site.LanguagePrefix }}/howto" target="_blank">google location file</a> (recommended)
+        </label>
+        <div class="question">
+            <label for="file">{{ i18n "select_location_file" }} <b>Location History.json</b></label>
+            <div class="options">
+                <input class="form-control" type="file" id="file" name="file">
+            </div>
+            <p id="datafilterinformation"></p>
         </div>
-        <p id="datafilterinformation"></p>
-    </div>
 
-</div>
+        <label>
+            <b>OR</b>
+        </label>
 
-        <div class="questionblock hidden" id="response">
-            <label>
-                <div id="uploadstatus" class="informationbox"></div>
-            </label>
-
-            <div class="question">
-                <div id="uploadinformation" class="informationbox"></div>
-            </div>
+        <label>
+            Manually select the places you visited in the past 15 days
```

Comment:
i18? I would add `(just klick on the map to add places)`

Reply:
yup. will do